conda-forge.yml

```
@@ -12,6 +12,8 @@ conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
+os_version:
+  linux_64: cos7
```

Even though this is already the default in the other cases (and soon will be for `linux-64`), maybe we should note these explicitly in all cases?

```suggestion
  linux_64: cos7
  linux_aarch64: cos7
  linux_ppc64le: cos7
```

We could also then drop these lines from `conda_build_config.yaml` as `os_version` also configures `c_stdlib_version` for us

https://github.com/conda-forge/openmpi-feedstock/blob/99059c0e090c0f8b6f821cfd5e80c0b1c35f62f0/recipe/conda_build_config.yaml#L1-L2